docs/DeveloperGuide.md

```diff
@@ -890,6 +890,48 @@ testers are expected to do more *exploratory* testing.
     3. If the data is beyond repair, delete the entire `docs` folder and the `AddressBook.json` file to start afresh.
 
 
+---
+
+## Appendix: Efforts
+
+WedLinker is a project adapted from the original [AddressBook3 (AB3)](https://se-education.org/addressbook-level3/) customized specifically for wedding planners by our team of five.
+Throughout this project, we had split the work evenly and adhering to the principles and concepts that were taught by the
+CS2103T teaching team, including:
+- [Forking Workflow](https://nus-cs2103-ay2425s1.github.io/website/schedule/week7/topics.html#project-management-revision-control-forking-flow)
+- [Feature Branch Flow](https://nus-cs2103-ay2425s1.github.io/website/schedule/week7/topics.html#project-management-revision-control-feature-branch-flow)
+- [UML Class Diagrams](https://nus-cs2103-ay2425s1.github.io/website/se-book-adapted/chapters/uml.html#class-diagrams)
+- [Testing](https://nus-cs2103-ay2425s1.github.io/website/se-book-adapted/chapters/testing.html)
+
+As we envisioned an application for wedding planners, we concocted multiple features that wedding planners we interviewed
+mentioned they wanted. These are features we have implemented to the WedLinker you see today.
+
+#### Difficulty Level:
+Adapting from the original AB3 to the WedLinker has been difficult as we opted for a more daring approach to implement 
+many features that wedding planners wanted. As with any software, the more features implemented, the more bugs there are,
+and WedLinker was no exception to it. 
+
+As a result, the difficulty of developing WedLinker came not only from the
+myriad of features we had to implement (and alter from the original AB3), but to also ensure the minimal number of
+bugs created.
```

Comment:
Maybe instead of 

>  but to also ensure the minimal number of bugs created.

you can say

> but to also ensure that there were minimal bugs created.